src/transform/FooterMenu.js

```diff
@@ -0,0 +1,219 @@
+import './FooterMenu.css'
+import Polyfill from './Polyfill'
+
+/**
+ * @typedef {function} FooterMenuItemPayloadExtractor
+ * @param {!Document} document
+ * @return {!Array.<string>} Important - should return empty array if no payload strings.
+ */
+
+/**
+ * @typedef {function} FooterMenuItemClickCallback
+ * @param {!Array.<string>} payload Important - should return empty array if no payload strings.
+ * @return {void}
+ */
+
+/**
+ * @typedef {number} MenuItemType
+ */
+
+// eslint-disable-next-line
```

Comment:
Please limit this to the rules you want to disable. e.g., `eslint-disable-next-line valid-jsdoc`.